source/chapters/1/security.md

```diff
@@ -78,32 +73,32 @@ export default class Security {
             throw new Meteor.Error('not-authorized', 'You are not authorized');
         };
     }
-    
     // add other business logic checks here that you use throughout the app
     // something like: isUserAllowedToSeeDocument()
-    // always keep decoupling your code if this class gets huge.
 }
 ```
+Word of advice: always keep decoupling your code if classes get huge. And by decoupling we mean that you should separate 
```

Comment:
I wouldn't redirect to the wikipedia website. User should be focused on Meteor and learning this. I suggest keep the first part, until you say. "Because of this..."